ofrak_patch_maker/ofrak_patch_maker_test/toolchain_c.py

```diff
@@ -169,3 +169,83 @@ def run_hello_world_test(toolchain_under_test: ToolchainUnderTest):
 
     assert os.path.exists(exec_path)
     assert get_file_format(exec_path) == tc_config.file_format
+
+
+def run_relocatable_test(toolchain_under_test: ToolchainUnderTest):
+    """
+    Use patchmaker to link on a relocatable binary
+    """
+    source_dir = os.path.join(os.path.dirname(__file__), "example_5")
+    source_path = os.path.join(source_dir, "patch.c")
+    build_dir = tempfile.mkdtemp()
```

Comment:
This should probably use the [pytest `tmpdir` fixture](https://docs.pytest.org/en/6.2.x/tmpdir.html).